discord/emoji.py

```diff
@@ -106,8 +106,8 @@ def created_at(self) -> datetime:
     @property
     def url(self) -> str:
         """Returns the URL of the emoji."""
-        fmt = "gif" if self.animated else "png"
-        return f"{Asset.BASE}/emojis/{self.id}.{fmt}"
+        url = f"{Asset.BASE}/emojis/{self.id}.{self.extension}"
+        return (url + "?animated=true") if self.animated else url
```

Comment:
```suggestion
		if self.animated:
		    url += "?animated=true"
        return url
```

More readable and consistent with the fact that we created a url variable there